frontend/src/lang/es.ts

```diff
@@ -43,6 +43,7 @@ export const es: LanguageConfig = {
         RSV: "VRS",
         HIV: "VIH",
         SYPHILIS: "Sífilis",
+        HEPATITIS_C: "Hepatitis C",
```

Comment:
should there be a dash in this string so it matches the name in other places?

Reply:
Checked with Jayna on this and we should actually switch everything else to use no dash so that it matches how the CDC refers to it as "Hepatitis C"

Reply:
@mehansen any preference for whether I make those naming changes in this current PR? I created a db changeset but debating whether to have it be a separate PR for simpler code review

Reply:
[Follow up PR created here](https://github.com/CDCgov/prime-simplereport/pull/8255)